README.md

```
@@ -39,31 +45,37 @@ dependencies {
     private Long zooId;
 }
-
 ```
 
 ---
 
 ### 3. 스냅샷 생성
 
-빌드하면 `build/classes/java/main/jinx/` 경로에 스키마 스냅샷이 생성됩니다.
+빌드 수행 시 다음 경로에 스냅샷이 자동 생성됩니다
 
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)